cortex-mixin/dashboards/writes.libsonnet

```diff
@@ -140,70 +140,6 @@ local utils = import 'mixin-utils/utils.libsonnet';
         utils.latencyRecordingRulePanel('cortex_kv_request_duration_seconds', $.jobSelector($._config.job_names.ingester))
       )
     )
-    .addRowIf(
-      std.member($._config.storage_engine, 'chunks'),
-      $.row('Memcached')
-      .addPanel(
-        $.panel('Requests / sec') +
-        $.qpsPanel('cortex_memcache_request_duration_seconds_count{%s,method="Memcache.Put"}' % $.jobMatcher($._config.job_names.ingester))
```

Comment:
Using memcached doesn't seem to rely on chunks storage. Perhaps we can keep it in?

Reply:
ingesters used to use a write-caching in memcache. This no longer exists in blocks. 

We do have memcache panels in the reads.libsonnet. Those stay.